trappy/ftrace.py

```diff
@@ -43,6 +44,37 @@ def _plot_freq_hists(allfreqs, what, axis, title):
         trappy.plot_utils.plot_hist(allfreqs[actor], ax, this_title, "KHz", 20,
                              "Frequency", xlim, "default")
 
+def _trace_cache_path(trace_file):
+    cache_dir  = '.' +  os.path.basename(trace_file) + '.cache'
+    cache_path = os.path.dirname(trace_file) +  '/' + cache_dir
```

Comment:
found why it broke them, because I wasn't using abspath causing mkdir to fail, fixed it in next rev.